### ISSUE TITLE
ADLSGen2 or Blob storage?

### ISSUE THREAD

**Issue by rasghub** [Mmodarre/AzureDataFactoryHOL#15]
The High Level Architecture visual indicates Storage Account v2 (ADLSGen2), however the environment preparation steps indicate a storage account without the Hierarchical namespace option selected (that would make it ADLS Gen2). 

Reviewing the steps, it seems that only blob storage is being used and not ADLSGen2.

**Comment by Mmodarre** [comment]
Storage Account V2 is different from ADLS G2. At this stage the lab is using Blob Storage to save on cost and brevity of instructions.

**Comment by rasghub** [comment]
I agree. I was just pointing out that the diagram says ADLSGen2 while the instructions use blob storage. Maybe consider removing ADLSGen2 from the diagram, to eliminate confusion.

On May 21, 2020, at 7:10 PM, Mmodarre <notifications@github.com> wrote:

﻿

Storage Account V2 is different from ADLS G2. At this stage the lab is using Blob Storage to save on cost and brevity of instructions.

**Comment by Mmodarre** [comment]
You are right the diagram is confusing. It was meant to comply Blob storage or ADLS Gen2 could be used in the solution. Thanks for pointing it out. I will reopen this to fix the diagram.
